src/components/header/drawer/DrawerSections.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC } from 'react';
+import { useI18n } from '@smg-automotive/i18n-pkg';
+import { Box, GridItem } from '@chakra-ui/react';
+
+import MobileOnlyAccordionPanel from 'src/components/mobileOnlyAccordion/MobileOnlyAccordionPanel';
+import MobileOnlyAccordionItem from 'src/components/mobileOnlyAccordion/MobileOnlyAccordionItem';
+import MobileOnlyAccordionButton from 'src/components/mobileOnlyAccordion/MobileOnlyAccordionButton';
+import MobileOnlyAccordion from 'src/components/mobileOnlyAccordion';
+import List from 'src/components/list';
+
+import { NavigationLinkNode } from '../config/drawerNodeItems';
+
+import { DrawerNavigationLink } from '.';
+
+export const NonCollapsibleSection: FC<{ node: NavigationLinkNode }> = ({
+  node,
+}) => {
+  return (
+    <Box py={{ base: '2xl', md: 0 }} px={{ base: '2xl' }}>
```

Comment:
If you just need one value then you can go for:
```suggestion
    <Box py={{ base: '2xl', md: 0 }} px='2xl'>
```
Or maybe you forgot to add another value for another breakpoint